Automattic/crowdsignal-forms
# Prepare release 1.5.6

This PR bumps version to 1.5.6

## Test instructions
Checkout and `make client`. Verify the version number and test the poll blocks: placing a vote should first do a preflight request to poll.fm/n/...
